org.civicrm.reqangular/src/common/decorators/angular-date/datepicker-addon.js

```diff
@@ -1,35 +1,88 @@
-define([], function () {
+/* eslint-env amd */
```

Comment:
also, can you please make it a separate decorator? let's not put all the decorations in one basket

Reply:
Done